about.html

```diff
@@ -0,0 +1,141 @@
+            </ul>
+        </nav>
+    </div>
+    </div>
+        <div class="bars" id="hambug">
+        <div class="hambuger"></div>
+        <div class="hambuger"></div>
+        <div class="hambuger"></div>
+        </div>
+
+        <div class="toggle-menu">
+            <div><img src="../images/togclose.png" alt="close toggle" id="togclose" ></div>
+
+            <div class="hambug-items">
+            <ul class="menu">
+                <li class="list-menu"><a href="about.html" class="a-link">About</a></li>
+                <li class="list-menu"><a href="#" class="a-link">Program</a></li>
+                <li class="list-menu"><a href="#" class="a-link">Join</a></li>
+                <li class="list-menu"><a href="#" class="a-link">Sponsor</a></li>
+                <li class="list-menu"><a href="#" class="a-link">News</a></li>
+                <li class="list-menu"><a href="#" class="a-link">CC Campaign</a></li>
+            </ul>
+            </div>
+
+    </div>
+    </header>
+
+    <section class="hero">
+        <div class="container intro">
+        <!-- <h1>"Hello! Africa"</h1> -->
+        <h2>AFRICA TECH SUMMIT 2022</h2>
+        <P class="description">Africa Tech Summit Nairobi connects tech leaders from the African ecosystem and international players under one roof. Network with key stakeholders including tech corporates, mobile operators, fintechs, DeFi & crypto ventures, investors, leading start-ups, regulators and industry stakeholders driving business and investment forward.</P>
+        
+        <P>Please contact us via Emial for further questions <br> concerning Africa Tech Summit 2022</P>
+
+        <p><span>africatechsummit2022@afsummit.org</span></p>
+        </div>
+    </section>
+
+    <section class="summit-logo">
+        <div class="summit-head">
+            <h2>Africa Tech<br> Summit 2022 Logo</h2>
+            <hr>
+        </div>
+        <p>The logo for Africa Tech Summit 2022 was decided through the logo competition from 8 June to 7 July</p>
+        <div class="afrilogo"><img src="./images/tech-logo.png" alt="Africa Tech summit logo"></div>
+    </section>
+
+    <section class="past-events">
+        <div class="past">
+        <h2>See Past Africa Tech Summits</h2>
+        <hr>
+        </div>
+        <div class="time"><p>Take a look at the last two Africa Tech summit that took place in Kenya and Rwanda</p></div>
+    
+        
+
+        <div class="event-container">
+            <div class="second-event">
+                <div class="event-content">
+                    <h2 class="year">2021</h2>
+                    <p class="city">Africa Tech Summit 2021 in Kigali</p>
+                </div>
+
+            </div>
+            <div class="first-event">
+                <div class="event-content">
+                    <h2 class="year">2020</h2>
+                    <p class="city"> Africa Tech Summit 2020 in Nairobi</p>
+                </div>
+            </div>
```

Comment:
- [ ]  Your Past Africa Tech Summits's section looks amazing. :100:  , but  I think you can inprove it when you use the same background-color just as the template [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
* Your Design point_down

![3333](https://user-images.githubusercontent.com/57562869/148466011-b9d23144-0ae6-49a7-ba47-6f61b5458a66.png)

* Original Design point_down
![444](https://user-images.githubusercontent.com/57562869/148466198-71f83c7c-6377-4c51-b9d0-f625f75a0889.png)

Reply:
I have done this.